src/Link.cc

```diff
@@ -563,6 +590,57 @@ Errors Link::ResolveInertial(
   return errors;
 }
 
+/////////////////////////////////////////////////
+void Link::CalculateInertials(sdf::Errors &_errors, const ParserConfig &_config)
+{
+  if (this->dataPtr->sdf->HasElement("inertial"))
```

Comment:
We should avoid checking `dataPtr->sdf` here. It could be a `nullptr` if the object was constructed programmatically.